#758 Add Echorouk

link8k opened the pull request:
from https://www.ennaharonline.com/live/
@themak1985 requested in #758

https://streamtest.in/logs/http-echorouk-live-tv-dzsecurity-net-8081-echo-echorouknews-playlist-m3u8
https://streamtest.in/logs/http-echorouk-live-tv-dzsecurity-net-8081-echo-echorouknews-playlist-m3u8

link8k committed:
#758 Add Echorouk 

from https://www.ennaharonline.com/live/
@themak1985 requested in #758

adamo331 commented on channels/dz.m3u:
```diff
@@ -5,3 +5,9 @@ http://178.132.3.162:88/canal_algerie/video.m3u8?token=test
 http://numidiatv-live.dzsecurity.net:8081/entv/EnnaharTV_SD/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Echorouk TV
+#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net
+https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8
```
not working

adamo331 commented on channels/dz.m3u:
```diff
@@ -5,3 +5,9 @@ http://178.132.3.162:88/canal_algerie/video.m3u8?token=test
+https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="News",Echorouk News
+#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net
+https://echorouk-live-tv.dzsecurity.net/echo/EchoroukNews/playlist.m3u8
```
not working

LaneSh4d0w commented:
DZ-Security got some weird things happening. It exists... but it's got some security issues due to some authenticity problems.

Funny thinking about a society that got "Security" inside of it.

LaneSh4d0w commented:
@link8k Change those with this, less techy and works : 

Echourouk TV : http://echorouk-live-tv.dzsecurity.net:8081/echo/EchoroukTV/playlist.m3u8
Echourouk News : http://echorouk-live-tv.dzsecurity.net:8081/echo/EchoroukNews/playlist.m3u8

link8k committed:
Switch to http

Thanks @LaneSh4d0w 
https://streamtest.in/logs/http-echorouk-live-tv-dzsecurity-net-8081-echo-echorouknews-playlist-m3u8
https://streamtest.in/logs/http-echorouk-live-tv-dzsecurity-net-8081-echo-echorouknews-playlist-m3u8

link8k commented:
Thanks @LaneSh4d0w @adamo331  Fixed
